vaccine/healthcheck_ussd.py

```diff
@@ -0,0 +1,798 @@
+        return MenuState(
+            self,
+            question="\n".join(
+                [
+                    "Please confirm the address below based on info you shared:",
+                    f"{{ address }}",
+                    "",
+                    "Reply",
+                ]
+            ),
+            choices=[
+                Choice("state_place_details_lookup", "Yes"),
+                Choice("state_city", "No"),
+            ],
+        )
+
+    async def state_place_details_lookup(self):
+        for i in range(3):
+            try:
+                response = await get_google_api().get(
+                    "https://maps.googleapis.com/maps/api/place/details/json",
+                    params={
+                        "key": config.GOOGLE_PLACES_KEY,
+                        "place_id": self.user.answers.get("place_id"),
+                        "sessiontoken": self.user.answers.get("google_session_token"),
+                        "language": "en",
+                        "fields": "geometry",
+                    },
+                )
+                response.raise_for_status()
+                data = await response.json()
+
+                if data["status"] != "OK":
+                    return await self.go_to_state("state_city")
+
+                location = data["result"]["geometry"]["location"]
+
+                lat = self.pad_location(location["lat"], 2)
+                lng = self.pad_location(location["lng"], 3)
+
+                self.save_answer("city_location", f"{lat}{lng}/")
+
+                if self.user.answers.get("confirmed_contact"):
+                    return await self.go_to_state("state_tracing")
+                return await self.go_to_state("state_age")
+            except aiohttp.ClientError as e:
+                if i == 2:
+                    logger.exception(e)
+                    return await self.go_to_state("state_error")
+                else:
+                    continue
+
+    async def state_age(self):
+        if self.user.answers.get("state_age"):
+            return await self.go_to_state("state_fever")
+
+        return ChoiceState(
+            self,
+            question="How old are you?",
+            error="\n".join(
+                [
+                    "Please use numbers from list.",
+                    "",
+                    "How old are you?",
+                ]
+            ),
+            choices=[
+                Choice("<18", "<18"),
+                Choice("18-40", "18-39"),
+                Choice("40-65", "40-65"),
+                Choice(">65", ">65"),
+            ],
+            next="state_fever",
+        )
+
+    async def state_age_years(self):
+        if self.user.answers.get("state_age") and self.user.answers.get(
+            "state_age_years"
+        ):
+            return await self.go_to_state("state_province")
+
+        def validate_age(content):
+            age = int(content)
```

Comment:
Need to manage the case where this is `None` or where it can't be converted to an integer